packages/blockchain-api/src/blockscout.ts

```diff
@@ -25,6 +26,8 @@ import { TransferCollection } from './transaction/TransferCollection'
 import { TransfersNavigator } from './transaction/TransfersNavigator'
 import { ContractAddresses, getContractAddresses } from './utils'
 
+const { performance } = require('perf_hooks')
```

Comment:
nit: This might be necessary for other reasons, but we do most imports with the ESM syntax `import { performance } from 'perf_hooks'` so we should probably use that for consistency unless there's a good reason not to